web/app.js

```diff
@@ -2761,7 +2761,8 @@ function webViewerWheel(evt) {
       let ticks = 0;
       if (
         deltaMode === WheelEvent.DOM_DELTA_LINE ||
-        deltaMode === WheelEvent.DOM_DELTA_PAGE
+        deltaMode === WheelEvent.DOM_DELTA_PAGE ||
+        window.chrome
       ) {
```

Comment:
This will bundle useless code in the Firefox PDF Viewer, which we always want to avoid.
```javascript
      if (
        (typeof PDFJSDev !== "undefined" &&
          PDFJSDev.test("GENERIC || CHROME") &&
          window.chrome) ||
        deltaMode === WheelEvent.DOM_DELTA_LINE ||
        deltaMode === WheelEvent.DOM_DELTA_PAGE
      ) {
```

Reply:
I'm confused, are you saying the window.chrome is useless due to the existence of the PDFJSdev.test command? But clearly the pdfjsdev command isn't doing its job properly if its not detecting the chromium browser

Reply:
I'm simply saying that you'll need to make exactly the changes mentioned above to avoid an unnecessary piece of code ending up in the *built-in* Firefox PDF Viewer.

Reply:
So I tried out the above change...but then the fix doesn't work anymore. I checked the debugger and that PDFJSDev variable doesnt even exist so maybe thats why the condition fails there itself. In all Chromium browsers it skips past that if statement and goes directly into the corresponding else statement (aka the pixel based calculation), thereby persisting the issue. So that above change doesnt help I think.

Reply:
> So I tried out the above change...but then the fix doesn't work anymore.

Yes it will, once you've built the code.

Reply:
You were right... I built and tried it out on chrome and it works like a charm. Updated my commit message accordingly.